packages/kit/src/views/DeviceManagement/pages/HardwareTroubleshootingModal/index.tsx

```
@@ -225,13 +225,21 @@ function HardwareTroubleshootingModal() {
                 height="auto"
               >
                 <YStack
+                  role="button"
                   f={1}
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)